Dockerfile

```diff
@@ -35,14 +44,11 @@ RUN microdnf install ca-certificates vi --nodocs &&\
     mkdir /licenses &&\
     microdnf clean all
 
+COPY --from=builder /go/src/github.com/open-cluster-management/insights-client/main /bin/main
+
 ENV VCS_REF="$VCS_REF" \
     USER_UID=1001
 
-
-
-ADD output/insights-client /bin
-
-
 EXPOSE 3030
 USER ${USER_UID}
-ENTRYPOINT ["/bin/insights-client"]
+ENTRYPOINT ["/bin/main"]
```

Comment:
What is the rationale behind switching our Dockerfile entrypoints from _project name_ to _main_ ? I appreciate the uniformity, just curious.

Reply:
just for consistency